dynamic-layers/openembedded-layer/recipes-devtools/bpftrace/bpftrace_0.21.2.bb

```diff
@@ -19,14 +19,10 @@ RDEPENDS:${PN} += "bash python3 xz"
 PV .= "+git"
 
 SRC_URI = "git://github.com/iovisor/bpftrace;branch=master;protocol=https \
-           file://0001-replace-python-with-python3-in-the-test.patch \
-           file://0002-ast-Repace-getInt8PtrTy-with-getPtrTy.patch \
-           file://0003-ast-Adjust-to-enum-changes-in-llvm-18.patch \
-           file://0004-cmake-Bump-max-LLVM-version-to-18.patch \
-           file://0001-use-64bit-alignment-for-map-counter-atomic-add.patch \
            file://run-ptest \
+           file://0001-cmake-Bump-max-LLVM-version-to-19.patch \
 "
```

Comment:
Do you happen to have buildpaths disabled or lowered to WARN_QA?

With ptest enabled I'm seeing (with older 0.20.1 version):
http://errors.yoctoproject.org/Errors/Build/190235/
```
ERROR: bpftrace-0.20.1+git-r0 do_package_qa: QA Issue: File /usr/lib/bpftrace/ptest/tests/bpftrace_test in package bpftrace-ptest contains reference to TMPDIR [buildpaths]
ERROR: bpftrace-0.20.1+git-r0 do_package_qa: QA Issue: File /usr/lib/bpftrace/ptest/tests/testprogs/cmake_install.cmake in package bpftrace-ptest contains reference to TMPDIR [buildpaths]
ERROR: bpftrace-0.20.1+git-r0 do_package_qa: QA Issue: File /usr/lib/bpftrace/ptest/tests/testprogs/CTestTestfile.cmake in package bpftrace-ptest contains reference to TMPDIR [buildpaths]
ERROR: bpftrace-0.20.1+git-r0 do_package_qa: QA Issue: File /usr/lib/bpftrace/ptest/tests/testlibs/cmake_install.cmake in package bpftrace-ptest contains reference to TMPDIR [buildpaths]
ERROR: bpftrace-0.20.1+git-r0 do_package_qa: QA Issue: File /usr/lib/bpftrace/ptest/tests/testlibs/CTestTestfile.cmake in package bpftrace-ptest contains reference to TMPDIR [buildpaths]
ERROR: bpftrace-0.20.1+git-r0 do_package_qa: QA Issue: File /usr/src/debug/bpftrace/0.20.1+git/parser.tab.hh in package bpftrace-src contains reference to TMPDIR [buildpaths]
ERROR: bpftrace-0.20.1+git-r0 do_package_qa: QA Issue: File /usr/src/debug/bpftrace/0.20.1+git/lex.yy.cc in package bpftrace-src contains reference to TMPDIR [buildpaths]
ERROR: bpftrace-0.20.1+git-r0 do_package_qa: QA Issue: File /usr/src/debug/bpftrace/0.20.1+git/parser.tab.cc in package bpftrace-src contains reference to TMPDIR [buildpaths]
ERROR: bpftrace-0.20.1+git-r0 do_package_qa: QA Issue: File /usr/src/debug/bpftrace/0.20.1+git/location.hh in package bpftrace-src contains reference to TMPDIR [buildpaths]
```

Reply:
yes in yoe `buildpaths` is downgraded to be a warning.